src/references/media_library.md

```diff
@@ -0,0 +1,88 @@
+---
+id: media_library_reference
+name: Media Library
+title: Media Library
+tags:
+  - Reference
+---
+
+# Media Library
+
+## Overview
```

Comment:
One thing that took me a second to realize was the difference between the raw image and "brushes". Could use a sentence of explaining the two.

Reply:
Yeah I did read that stuff and think it's a little confusing so kept it simple. Will consider adding a note about it.